### ISSUE TITLE
Confusion about next and current event

### ISSUE THREAD

**Issue by richardboehme** [rughh/on_ruby#1084]
Hey! We've just published our next meetup which is still some time away (more than 3 months ago). After our current event was over I was expecting that the next event will be "promoted" on the homepage of the user group. However, it does not show the next event but the custom recurrence text of the user group.

I've already inspected the code on why this is happening and I'm not sure if I understand the reasoning behind the implementation. Basically we look for the "current event" which is implemented using the [`current` scope](https://github.com/rughh/on_ruby/blob/master/app/models/event.rb#L25) in the `Event` model. This scope specifically looks for events that take place in the next 9 weeks. The number feels quite arbitrary to me and I feel like just changing this to display the next planned event (with min date in the future) should work best for frequent and less frequent meetups. But maybe there is a special reason why we want to enforce a specific actuality?

Another option I could think of is to let admins control which event should be "promoted" on the home page, however this might become tedious to manage and would need some UI on the admin page.

Let me know what you think and if you're open to change the behavior.

**Comment by phoet** [comment]
I guess that this was tailored to our Meetup schedule, but not sure exactly.

**Comment by richardboehme** [comment]
> I guess that this was tailored to our Meetup schedule, but not sure exactly.

Yeah I guess so, it was also changed once because the cologne.rb meeting was 9 weeks in the future :D
